apps/web/src/pages/templates/TemplatesListPage.tsx

```diff
@@ -31,6 +51,39 @@ function NotificationList() {
     navigate(ROUTES.TEMPLATES_CREATE);
   };
 
+  const handleTryDigestClick = () => {
```

Comment:
when the `Try Digest Workflow` button is clicked we do create an example Digest Workflow

Reply:
Not sure if this should be driven (the example workflow content) by the backend. 🤔

Reply:
we do the same for the in-app workflow and I think that for now, it's ok to have it like this... 
I feel like we should use what is available to accomplish the goal and not overengineer with the FE needs...